src/backend/dx11/src/lib.rs

```diff
@@ -978,12 +986,17 @@ impl queue::CommandQueue<Backend> for CommandQueue {
         _image: ImageView,
         _wait_semaphore: Option<&Semaphore>,
     ) -> Result<Option<window::Suboptimal>, window::PresentError> {
-        surface
-            .presentation
-            .as_ref()
-            .unwrap()
-            .swapchain
-            .Present(1, 0);
+        let mut presentation = surface.presentation.as_mut().unwrap();
+        let (interval, flags) = match presentation.mode {
+            window::PresentMode::IMMEDIATE => (0, DXGI_PRESENT_ALLOW_TEARING),
+            //Note: this ends up not presenting anything for some reason
+            //window::PresentMode::MAILBOX if !presentation.is_init => (1, DXGI_PRESENT_DO_NOT_SEQUENCE),
+            window::PresentMode::MAILBOX |
+            window::PresentMode::FIFO => (1, 0),
+            _ => (0, 0),
+        };
+        presentation.is_init = false;
```

Comment:
this is only for the sequencing part?

Reply:
this is related to the `Note` above, for some reason I wasn't able to make it work
with the line commented out, the `is_init` isn't used, but it's fine